### DISCUSSION TITLE
Multi-module maven project with jar and war packing

### DISCUSSION THREAD

**Discussion by 45jjmalaluan** [CycloneDX/cyclonedx-maven-plugin#498] [Q&A]
Hi,

Using cyclonedx-maven-plugin version 2.8.0.

In the root pom.xml,
```
				<executions>
					<execution>
						<phase>package</phase>
						<goals>
							<goal>makeAggregateBom</goal>
						</goals>
					</execution>
				</executions>
```

When I run mvn clean package.

In the sub-module 1  with packaging=jar

I see in the bom.json. Seems to be fine.

```
    "properties" : [
      {
        "name" : "maven.goal",
        "value" : "makeBom"
      },
```

In the sub-module 2  with packaging=war

I see in the bom.json.

```
    "properties" : [
      {
        "name" : "maven.goal",
        "value" : "makeBom"
      },
```

Is that correct? I was assuming to see `makePackageBom `.

**Comment by tib-rijo** [comment]
It seems makePackageBom will take care of running multiple makeBom targets for you.

**Comment by albogdano** [comment]
Yes, `makePackageBom` is the way to go. https://stackoverflow.com/questions/76999471/how-can-i-generate-sbom-in-sub-module-and-get-an-aggregated-sbom-regression-wit/79882741#79882741
